tests/drivers/stepper/stepper_api/Kconfig

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2024 Navimatix GmbH
+# SPDX-License-Identifier: Apache-2.0
+
+config ENTROPY_GENERATOR
+	depends on ZTEST_SHUFFLE
+	bool "Shuffle the order of tests and suites"
+	default y
```

Comment:
This looks like its reverse - shuffling would depend on the entropy generator no?